docs/notebooks/first_tutorial.jl

```diff
@@ -406,12 +439,32 @@ md"""
 	The `:metadata` target is special, since it doesn't write to CSV but to TOML. Add the `:metadata` target to the export list and inspect the result.
 """
 
+# ╔═╡ adf67033-5941-4be6-bb17-c0958348b905
+md"""
+## Plot the extracted data
+"""
+
+# ╔═╡ e26defa5-10ff-4275-bae9-768f7fb8d9ba
+md"""
+We will use Package Dataframes to process the csv file, and again use Makie to visualise the data.
+"""
+
+# ╔═╡ e17f35e7-8d09-4da1-880f-563bc49b364c
+md"""
+Using the follwing command to read the csv file
+"""
+
 # ╔═╡ 329d30f1-797e-4522-9c20-e60d35079f5f
 # ╠═╡ disabled = true
 #=╠═╡
 adm = read_csv("$(OUTPUTDIR)/tutorial_adm.csv", DataFrame)
   ╠═╡ =#
 
+# ╔═╡ add6a25b-d948-4cd0-9412-56752793ca4b
```

Comment:
I cannot run the code cell with the ADM, it says "This cell depends on a distabled cell" but I think I enabled all of them.

Reply:
Seems like a hiccup in Pluto. The cell depends on the cell above; after enabling that one I had to click the little play button at the bottom of the cell again to make it run again.